test/test.sh

```diff
@@ -25,7 +25,7 @@ if [ -z "$TEST_BUCKET_PREFIX" ]; then
 fi
 
 if [ -z "$S3_COMMAND" ]; then
-    S3_COMMAND=s3
+    S3_COMMAND="s3 -u"
```

Comment:
Why are you changing this?  And without updating the comment at the beginning of the file that documents what the default value is?